src/BUILD

```diff
@@ -752,6 +752,9 @@ filegroup(
         "@openjdk14_darwin_archive//:WORKSPACE",
         "@openjdk14_linux_archive//:WORKSPACE",
         "@openjdk14_windows_archive//:WORKSPACE",
+        "@openjdk15_darwin_archive//:WORKSPACE",
```

Comment:
Please add also
```        
        "@remotejdk15_linux_for_testing//:WORKSPACE",
        "@remotejdk15_macos_for_testing//:WORKSPACE",
        "@remotejdk15_win_for_testing//:WORKSPACE",
```

to test_repos filegroup.

Reply:
Done, thanks!